app/assets/v2/js/pages/carousel.js

```diff
@@ -1,28 +1,27 @@
-const slides = document.querySelectorAll('#slides .slide');
+const slides = document.querySelectorAll( '#slides .slide' );
```

Comment:
it might be by preference, but I haven't seen this kind of style before. (add blank before & after function parameter)

Reply:
Yea it isn't a very common thing I think by most companies.  There have been other references to that style though as it promotes readability for the most part, at least IMO.

Also [jQuery](https://contribute.jquery.org/style-guide/js/#good-examples) uses it.